tool/teleport/common/teleport.go

```diff
@@ -346,7 +366,9 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	case status.FullCommand():
 		err = onStatus()
 	case dump.FullCommand():
-		err = onConfigDump(dumpFlags)
+		err = onConfigDump(dumpFlags, []string{})
+	case dumpNodeConfigure.FullCommand():
+		err = onConfigDump(dumpFlags, []string{"ssh"})
```

Comment:
I do wonder if doing something like this keeps it cleaner and avoids introducing the additional slice of strings that realistically conflicts with the `.Roles` field:

```suggestion
                dumpFlags.Roles = "node"
		err = onConfigDump(dumpFlags)
```

I'm not 100% attached to this, but it definitely feels like it might keep the logic clearer. I definitely had to think twice when I read `onConfigDump` to understand why there were two separate things influencing whether or not a field is included.

Reply:
I like this, good idea! 👍